### PR TITLE
[EOC-88] remove shopping from app

### PR DESCRIPTION
**Pull request by alemikolo**

### CODE REVIEW

File: client/src/modules/list/components/ItemsContainer.jsx [hunk 1]
```diff
@@ -113,26 +113,26 @@ class ProductsContainer extends Component {
...
 
-ProductsContainer.propTypes = {
+ItemsContainer.propTypes = {
   archived: PropTypes.bool,
   children: PropTypes.node,
   currentUser: PropTypes.objectOf(PropTypes.string).isRequired,
   description: PropTypes.string,
   name: PropTypes.string,
-  products: PropTypes.arrayOf(PropTypes.object)
+  items: PropTypes.arrayOf(PropTypes.object)
```
Comment (marekrozmus) [inline client/src/modules/list/components/ItemsContainer.jsx]: alphabeticla ordering

File: client/src/modules/list/components/ItemsList/ItemsList.jsx [hunk 2]
```diff
@@ -97,10 +97,10 @@ class ProductsList extends Component {
   }
 }
 
-ProductsList.propTypes = {
+ItemsList.propTypes = {
   currentUser: UserPropType.isRequired,
   match: RouterMatchPropType.isRequired,
-  products: PropTypes.arrayOf(PropTypes.object),
+  items: PropTypes.arrayOf(PropTypes.object),
```
Comment (marekrozmus) [inline client/src/modules/list/components/ItemsList/ItemsList.jsx]: alphabetical ordering

File: server/routes/item.js [hunk 3]
```diff
@@ -10,16 +10,16 @@ const {
 } = require('../controllers/item');
 const { authorize } = require('../middleware/authorize');
 
-// Add new product
+// Add new item
```
Comment (marekrozmus) [inline server/routes/item.js]: get rid of all these comments - they do not introduce any additional information that can be already read from code

File: client/src/modules/list/model/actions.js [hunk 4]
```diff
@@ -157,8 +157,8 @@ export const createList = (
 
 export const fetchListsMetaData = (cohortId = null) => dispatch => {
   const url = cohortId
-    ? `${ENDPOINT_URL}/shopping-lists/meta-data/${cohortId}`
-    : `${ENDPOINT_URL}/shopping-lists/meta-data`;
+    ? `${ENDPOINT_URL}/lists/meta-data/${cohortId}`
+    : `${ENDPOINT_URL}/lists/meta-data`;
   dispatch(fetchListsMetaDataRequest());
   getData(url)
```
Comment (adamklepacz) [inline client/src/modules/list/model/actions.js]: add return here just in case

File: server/controllers/list.js [hunk 5]
```diff
@@ -175,13 +175,13 @@ const getListData = (req, resp) => {
...
+        _id,
         adminIds,
         cohortId,
         description,
-        _id,
         isArchived,
-        name,
-        products
+        items,
+        name
```
Comment (adamklepacz) [inline server/controllers/list.js]: maybe sort alphabetically here

**Pull request merged**